src/graphql/queries/getGallery.js

```diff
@@ -0,0 +1,14 @@
+import { gql } from "@apollo/client";
```

Comment:
Just suggestion: to keep your modules as much isolated as it possible, it's better to create queries inside module folder. And move it to common only when you see, that it's common query. 
Also you should do it with constants, components and other stuff which refers only to this module (screen).